modules/vmseries/main.tf

```diff
@@ -9,6 +9,17 @@ locals {
     }
     if try(v.public_ip, null) != null || local.create_public_ip[k]
   }
+  create_public_ipv6 = {
+    for k, v in var.network_interfaces : k => try(v.create_public_ipv6, false)
+  }
+  ipv6_access_configs = {
+    for k, v in var.network_interfaces : k => {
+      external_ipv6               = try(v.public_ipv6, null) != null ? v.public_ipv6 : local.create_public_ipv6[k]
```

Comment:
As discussed, `local.create_public_ipv6[k]` is a boolean.

Reply:
This is an error on my side. Should be fixed now.